packages/inspection/src/database/InspectionApiWorker.ts

```diff
@@ -1,6 +1,6 @@
 import { InspectionContext } from "../main.js";
 
-const MAX_NUMBER_OF_IDS = 5000
+// Not used: const MAX_NUMBER_OF_IDS = 5000
```

Comment:
Should we remove it?

Reply:
Ok with me, but since you added this I was not sure you mighyt need it (again).
Removed this for now.